scripts/sbatch_eval/run_megatron_server_client.sbatch

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#SBATCH --job-name=eval-harness
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1          # crucial - only 1 task per dist per node!
+#SBATCH --cpus-per-task=48           # number of cores per tasks
+#SBATCH --hint=nomultithread         # we get physical cores not logical
+#SBATCH --time 02:00:00              # maximum execution time (HH:MM:SS)
+#SBATCH --account=opengptx-elm
+#SBATCH --partition=booster
+#SBATCH --output=logs/%x-%A_%a.out  # output file name
+#SBATCH --error=logs/%x-%A_%a.err   # error file name
+#SBATCH --array=0-26
```

Comment:
The array IDs would also need to be modified according to the length of the checkpoint list. Maybe it would make sense to write another small wrapper that automatically sets `SBATCH_ARRAY_INX="$(wc -l "$MODEL_CHECKPOINTS")"` and then `sbatch`es this script (with this line removed).

Reply:
I was thinking, passing `--array="$(wc -l "$MODEL_CHECKPOINTS")"` to `sbatch` using the wrapper would be more verbose and probably carry the intent across better.